component/client/RoomService.js

```diff
@@ -0,0 +1,33 @@
+import * as firebase from "firebase";
+
+export default class RoomService {
```

Comment:
Though, this line is actually from firecasts' tutorial. I don't think they recognize this convention.